frontend/src/api/quizApi.js

```diff
@@ -17,3 +17,21 @@ export const getQuizResult = async (quizId) => {
     return error.data
   }
 }
+
+export const postQuizCreate = async (content, option1, option2, option3, option4, answer) => {
+  try {
+    const response = await api.post('/quiz/create', {
+      content: content,
+      option1: option1,
+      option2: option2,
+      option3: option3,
+      option4: option4,
+      answer: answer,
+    })
```

Comment:
JavaScript에서 객체를 생성할 때 `Shorthand property names`라고 key를 따로 넣지 않으면 변수 이름이 그대로 들어가! 그리고 매개변수를 받을 때도 6개를 이렇게 받으면 순서를 헷갈리거나 의도하지 않은 값을 넣을 수 있어서 좋지 않아

처음부터 이렇게 객체로 받으면 간편하긴 하지만 객체에 어떤 값이 들어가야하는지 알 수 없고, 이상한 객체가 매개변수로 들어와도 거를 수 없기때문에 좋은 방법은 아니라고 생각해
```
export const postQuizCreate = async (quiz) => {
  try {
    const response = await api.post('/quiz/create', quiz)
```

그래서 이렇게 구조분해할당된 객체를 매개변수로 받으면 순서에 상관없이 해당되는 key의 값만 받을 수 있어
```suggestion
export const postQuizCreate 
    = async ({ content, option1, option2, option3, option4, answer }) => {
  try {
    const response = await api.post('/quiz/create', {
      content,
      option1,
      option2,
      option3,
      option4,
      answer,
    })
```
참고
[MDN Object Initializer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) 
[Airbnb Style Guide](https://github.com/airbnb/javascript?tab=readme-ov-file#destructuring)

Reply:
적용했습니다.